packages/testsuite/cypress/support/navigation.ts

```diff
@@ -46,6 +46,10 @@ Cypress.Commands.add("logoutFromWebConsole", () => {
   cy.get("#logout-link").click();
 });
 
+Cypress.Commands.add("confirmYesInWizard", () => {
```

Comment:
> confirmYesInWizard

-> confirmYesInModal ?

Reply:
we have also method like `confirmAddResourceWizard`, `confirmNextInWizard`, `confirmFinishInWizard`. They are very similar.  The difference is only the text on the button. We could create method like: `confirmModal(buttonText)`.  But this update will trigger changes in a lot of test cases. 

So maybe we can create a issue to remove these methods and replace them general one is a separate MR. WDYT?